### ISSUE TITLE
GUI no longer works with fresh CTF environment install

### ISSUE THREAD

**Issue by skliper** [nasa/CTF#28]
After a fresh run of setup_ctf_env.sh, conda is now 4.12.0, whereas 4.5.12 was formerly installed.  Not sure what dependency updates causes the issue, but the GUI will no longer open.  Part of the trace:

```
ctf/tools/ctf_ui/node_modules/pkg-dir/index.js:5
    const pkgDir = async cwd => {
                         ^^^

    SyntaxError: Unexpected identifier
        at NativeCompileCache._moduleCompile (ctf/tools/ctf_ui/node_modules/v8-compile-cache/v8-compile-cache.js:240:18)
        at Module._compile (ctf/tools/ctf_ui/node_modules/v8-compile-cache/v8-compile-cache.js:184:36)
        at Object.Module._extensions..js (module.js:586:10)
        at Module.load (module.js:494:32)
        at tryModuleLoad (module.js:453:12)
        at Function.Module._load (module.js:445:3)
        at Module.require (module.js:504:17)
```

@dmknutsen

**Comment by blueoceanwater** [comment]
My conda is 4.9.2, that version works

**Comment by blueoceanwater** [comment]
thanks for the update. Let me try it

**Comment by blueoceanwater** [comment]
It is completed and will be officially released in v1.6.
